connectors/totalenergies/src/index.js

```diff
@@ -258,15 +258,15 @@ class TemplateContentScript extends ContentScript {
     this.log('getContract starts')
     const contractElement = document.querySelector('.cadre2')
     const offerName = contractElement.querySelector('h2').innerHTML
-    const rawStartDate = contractElement.querySelector('p[class="txt-gras"]').innerHTML
+    const rawStartDate = contractElement.querySelector('p[class="font-700"]').innerHTML
     const splittedStartDate = rawStartDate.split('/')
     const day = splittedStartDate[0]
     const month = splittedStartDate[1]
     const year = splittedStartDate[2]
     const startDate = new Date(year, month, day)
     const href = contractElement.querySelectorAll('.ml-std')[1].children[1].getAttribute('href')
     const fileurl = `https://www.totalenergies.fr${href}`
-    const filename = `${day}-${month}-${year}_TotalEnergie_Contrat_${offerName.replaceAll(' ', '-')}.pdf`
+    const filename = `${year}-${month}-${day}_TotalEnergie_Contrat_${offerName.replaceAll(' ', '-')}.pdf`
```

Comment:
We should probably have a `normalizeFileName` function shared accross konnectors to avoid reinventing the wheel and remain consistent ;)

Reply:
Yup, this is planed, it should be added anytime from now :)